lib/datastore/dataset.js

```diff
@@ -78,6 +78,9 @@ var SCOPES = [
  *     `credentials` object.
  * @param {object=} options.credentials - Credentials object, used in place of
  *     a `keyFilename`.
+ * @param {string=} options.apiEndpoint - Override the default API endpoint used
+ *     to reach Datastore. This is useful for connecting to your local Datastore
+ *     server.
```

Comment:
Put an example here, preferably with the default port of whatever the local datastore runs on out of the box.